docs/dev-guides/personal-environments.md

```diff
@@ -469,6 +469,128 @@ gke-terra-integration-default-v2-95d71f95-gdfy   Ready    <none>   30h   v1.17.1
 
 </details>
 
+### Running Resiliency Tests Through Test Runner Library
+Test Runner Library provides the underlying framework for running resiliency tests within namespaces. The whole process involves only a few simple setup and configuration steps described below.
```

Comment:
define what a resiliency test is here

Reply:
Reorganized and add definition of resiliency test.